yarn-project/sequencer-client/src/simulator/public_executor.ts

```diff
@@ -167,3 +173,14 @@ export class WorldStateDB implements CommitmentsDB {
     return await this.db.findLeafIndex(MerkleTreeId.NOTE_HASH_TREE, commitment.toBuffer());
   }
 }
+
+/**
+ * Implements Nullifier db.
+ */
+export class NullifiersDB implements NDB {
```

Comment:
Is this functionally different to the commitments db interface, which is just meant to hold the append only trees?

Reply:
I didn't realize that getNullifierIndex was already present in DBOracle/SimulatorOracle. I put it in the CommitmentsDB interface and deleted NullifiersDB